sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -860,7 +860,7 @@ case class ShowTablesCommand(
   }
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
-    // Since we need to return a Seq of rows, we will call getTables directly
+    // Since we need to return a Seq of rows, we will call listTables directly
```

Comment:
Please revert the change in this PR. We usually do fix minor things when we need to touch that file.

Reply:
Sure, reverted the changes in 55a5997bdfe34439a171d3603fc98975b7c18129.